src/components/Card.js

```diff
@@ -31,51 +67,114 @@ const Wrapper = styled.div`
   padding: 15px;
   border-radius: 13px;
   box-shadow: 0px 0px 20px #dedede;
+  display: flex;
+  flex-direction: column;
+`;
+
+const Heading = styled.div`
```

Comment:
You ***could*** convert this `div`into a `header`so each card has its own actual header. That's totally optionally though.